src/utils/commentsApi.jsx

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+export const fetchCommentsById = (id) => {
+  return axios.get(`https://nc-news-xd0a.onrender.com/api/articles/${id}/comments`)
+    .then(response => response.data)
+    .catch(error => {
```

Comment:
This catch is in the middle of our promise chain and would therefore be better placed at the beginning/end of the promise chain